api/v1/manager_types.go

```diff
@@ -94,6 +166,94 @@ type ManagerList struct {
 	Items           []Manager `json:"items"`
 }
 
+func (c *Manager) GetMetadata() *Metadata {
+	return nil
+}
+
+func (c *Manager) GetMinReadySeconds() *int32 {
+	return nil
+}
+
+func (c *Manager) GetPodTemplateMetadata() *Metadata {
+	return nil
+}
+
+func (c *Manager) GetInitContainers() []v1.Container {
+	if c.Spec != (ManagerSpec{}) {
```

Comment:
I don't think we need this clause - `c.Spec` isn't a pointer, so we don't need to check for its existence.